### PR TITLE
rtmp-services: Update Switchboard Live servers

### PR DESCRIPTION
**Pull request by sarwin-github**
### Description
The Switchboard Live Server URLs have been recently updated in the application, so the URLs need to be updated in OBS to match.

### Motivation and Context
There were recent changes to the ingestion servers at Switchboard Live. We also removed Legacy url.

### How Has This Been Tested?
- This was tested using the latest build of OBS on Windows.
- Testing consisted of going live with every server to ensure connectivity was stable.

### Types of changes
- Tweak (non-breaking change to improve existing functionality). 
- Removed Alt names

### Checklist:

- [ ] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [ ] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [ ] I have included updates to all appropriate documentation.

### CODE REVIEW

File: plugins/rtmp-services/data/services.json
Comment (gxalpha) [inline plugins/rtmp-services/data/services.json:1156]: Is this changed intentional? This will break old setups that still have the old name set.

Comment (sarwin-github) [reply plugins/rtmp-services/data/services.json:1156]: Hello @gxalpha the `alt_names: "Switchboard Live (Joicaster)"` is no longer valid for Switchboard, so yes it's intentional.
Can you explain what are those breaking changes?  Thanks

Comment (gxalpha) [reply plugins/rtmp-services/data/services.json:1156]: The settings in OBS store the selected service by its name.
Any user that had selected "Switchboard Live (Joicaster)" before the name was changed to "Switchboard Live" in https://github.com/obsproject/obs-studio/commit/3ec053ef5df885571ba48aac6daf077412ed71b9 would still have that name stored in their OBS settings. OBS uses the `alt_names` to figure out what service that refers to today.
*As far as I know* (and I could be mistaken here), `alt_names` are not displayed anywhere in the UI.

Comment (sarwin-github) [reply plugins/rtmp-services/data/services.json:1156]: Got it, I might need to update my PR to simply remove the word (Joicaster) on the alt_names

Comment (Fenrirthviti) [reply plugins/rtmp-services/data/services.json]: The `alt_name` should *not* be updated to remove the previous name, as this may potentially break existing user's setups. Only the `name` value is shown in the UI, but alt_name is used to match configurations that are from before the change.

File: plugins/rtmp-services/data/services.json
Comment (gxalpha) [inline plugins/rtmp-services/data/services.json:1155]: The previous comment still has not been addressed, see https://github.com/obsproject/obs-studio/pull/10021#discussion_r1431835434.

**Review by gxalpha — CHANGES_REQUESTED** [review]
The commit history is getting ugly, try to only have one commit containing the change, without any merge commits. This is why it's recommended to not use the master branch for PRs.
See the comment below.

**Comment by Fenrirthviti** [comment]
Commits still need to be squashed before this can be merged.

File: plugins/rtmp-services/data/services.json
Comment (RytoEX) [reply plugins/rtmp-services/data/services.json:1155]: @sarwin-github This has still not been addressed. Do not change existing entries in alt_names. They are there for backwards compatibility reasons. We will not merge this as-is.

Comment (RytoEX) [reply plugins/rtmp-services/data/services.json]: I fixed this manually.

**Review by RytoEX — APPROVED** [review]
I've fixed the issues with this PR manually because the legacy ingest is gone and is causing CI failures when we try to run our services tests.

In the future:
1. Do not submit a PR from your fork's master branch.
2. Do not use merge commits to keep your branch up to date. Use `git rebase` instead.
3. Do not change or remove existing entries in the `alt_names` key. You can only add new entries. This is used so that existing user setups with the old names do not break.

If you do not follow this advice, your future PRs may be closed.

**Pull request merged**
